model/riscv_insts_zfa.sail

```diff
@@ -685,54 +685,57 @@ function clause execute(RISCV_FLTQ_D(rs2, rs1, rd)) = {
 
 /* FCVTMOD.W.D */
 
+/* Calculate the low 32 bits of the integer value from the binary64
+ * floating-point number and return them together with an indicator
+ * whether the NV (overflow) or NX (inexact) exceptions should be raised.
+ */
+val      fcvtmod_helper_inner : forall 'm, 'm >= 0. (bits(1), int('m), bits(53)) -> (bits(5), bits(32))
+function fcvtmod_helper_inner(sign, true_exp, true_mant) = {
+    /* perform the exponentation on the fixed-point
+       mantissa and extract the integer part */
+    let fixedpoint = to_bits(84, unsigned(true_mant)) << true_exp;
```

Comment:
Unnecessary round trips through mathematical integers here; should be able to stay in bitvector land